### PR TITLE
new records BDLaethg40

### PR DESCRIPTION
**Pull request by thea-m**
https://github.com/BetaMasaheft/Documentation/issues/2212

### CODE REVIEW

File: new/LIT6739PPDam.xml [hunk 1]
```diff
@@ -0,0 +1,71 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT6739PPDam" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="en" xml:id="t1">Protective prayer against hemorrhage (<foreign xml:lang="gez">dam</foreign>),
```
Comment (MassimoVilla) [inline new/LIT6739PPDam.xml:8]: What do you think about giving, in this particular case, `<foreign xml:lang="gez">ḥəmāma dam</foreign>` as equivalent to hemorrhage?

**Review by MassimoVilla — CHANGES_REQUESTED** [review]
Thank you Thea. I have just one question more than a remark

File: new/LIT6739PPDam.xml (hunk 1 above)
Comment (thea-m) [reply new/LIT6739PPDam.xml]: yes, that's good!

**Review by MassimoVilla — APPROVED** [review]

File: new/LIT6740PPWegat.xml [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+                <listRelation>
+                    <relation name="lawd:hasAttestation" active="LIT6740PPWegat" passive="BDLaethg40"/>
+                </listRelation>
+            </div>
+            <div type="edition">
+                <note>The following is the entire text of the prayer, taken from <ref type="mss" corresp="BDLaethg40"/>.</note>
+                <div type="textpart" xml:lang="gez">
+                    <ab>
+                        ጸሎት፡ በእንተ፡ ሕማመ፡ ውጋት፡ ወቁርጸት፡ ድሜጥርስ፡ ድሜጥርስ፡ ድሜጥርስ፡
+                        ጋይስጦስ፡ <gap reason="illegible"/>ስጦስ፡ ጋይስጦስ፡ <gap reason="illegible"/>ተው፡ ሰቁታው፡ ሰቅ<gap reason="illegible"/>ው፡
```
Comment (karljonaskarlsson) [inline new/LIT6740PPWegat.xml:66]: ታው፡ ?
ሰቈታው፡ ሰቆ

File: new/LIT6740PPWegat.xml [hunk 3]
```diff
@@ -0,0 +1,74 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT6740PPWegat" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="en" xml:id="t1">Protective prayer in Gǝʿǝz and Amharic against stomachache (<foreign xml:lang="gez">wəgʿāt</foreign>
```
Comment (karljonaskarlsson) [inline new/LIT6740PPWegat.xml:8]: wǝgʿat ?

**Comment by thea-m** [comment]
Thank you, I carried out the corrections. In the future, I think it is better when commenting on merged PRs to either 1) open a PR yourself with the suggested corrections or 2) open an issue to discuss them. Comments on merged PRs run a high risk of getting lost, because they are dependent on people staying on top of email notifications :)

**Pull request merged**